gadget/install/params.go

```diff
@@ -31,9 +30,14 @@ type Options struct {
 	Encrypt bool
 }
 
-type SystemInstallObserver interface {
-	gadget.ContentObserver
-	// ChosenEncryptionKey stores the encrypted data partition key to be sealed
-	// at the end of the installation process.
-	ChosenEncryptionKey(secboot.EncryptionKey)
+// EncryptionKeySet is a set of encryption keys.
+type EncryptionKeySet struct {
+	Key         secboot.EncryptionKey
+	RecoveryKey secboot.RecoveryKey
+}
+
+// InstalledSystemSideData carries the side data of an installed system.
```

Comment:
s/the side data/side data/

... of an installed system,  e.g.  secrets to access its partitions.